tachyon/math/polynomials/univariate/batch_fft_benchmark.cc

```diff
@@ -8,26 +8,30 @@ namespace tachyon::math::polynomials::univariate {
 
 constexpr size_t kMaxDegree = (size_t{1} << 32) - 1;
 template <typename F>
-void BM_BatchFFTSerial(benchmark::State& state) {
+void BM_SerialFFT(benchmark::State& state) {
```

Comment:
Can avoid intermediate change?

Reply:
Sorry this commit should be squashed with the original. So it will be removed.